subprojects/internal-integ-testing/build.gradle.kts

```diff
@@ -47,6 +47,7 @@ dependencies {
     implementation(project(":launcher"))
     implementation(project(":build-events"))
     implementation(project(":build-option"))
+    implementation(project(":toolchains-jvm"))
```

Comment:
Is this dependency still required now that `JavaInstallationRegistry` is not being moved?

Reply:
Looks like no.